GitHubRepositories.ps1

```diff
@@ -574,37 +577,24 @@ filter Get-GitHubRepository
             }
             elseif ([String]::IsNullOrWhiteSpace($RepositoryName))
             {
-                if ($PSCmdlet.ParameterSetName -eq 'ElementsOrUser')
-                {
-                    $telemetryProperties['UsageType'] = 'User'
-                    $telemetryProperties['OwnerName'] = Get-PiiSafeString -PlainText $OwnerName
+                $telemetryProperties['UsageType'] = 'User'
+                $telemetryProperties['OwnerName'] = Get-PiiSafeString -PlainText $OwnerName
 
-                    $uriFragment = "users/$OwnerName/repos"
-                    $description = "Getting repos for $OwnerName"
-                }
-                else
-                {
-                    $message = 'RepositoryName could not be determined.'
-                    Write-Log -Message $message -Level Error
-                    throw $message
-                }
+                $uriFragment = "users/$OwnerName/repos"
+                $description = "Getting repos for $OwnerName"
             }
             else
             {
-                if ($PSCmdlet.ParameterSetName -eq 'ElementsOrUser')
+                if ($PSBoundParameters.ContainsKey('Type') -or
```

Comment:
You've lost this same validation for the `Uri` case, which is also retrieving a specific repo.

Reply:
Done